frontEnd/src/components/App.jsx

```diff
@@ -0,0 +1,42 @@
+import React, { useState } from "react";
+
+import EmojiButton from "./EmojiButton/EmojiButton";
+import styles from "./App.module.css";
+
+function App() {
+  const handleButtonPress = label => {
+    updateSelectedMood(label);
+  };
+
+  const emojiButtonArray = [
+    { label: "🤠" },
```

Comment:
I realise that unicode characters are legitimate strings but I'd had a preference for storing "happy" or "sad" or "anxious" or something that is a bit more readable for everyone.